params/query_dev.yml

```diff
@@ -3,14 +3,21 @@ md_keywords:
 - molecular dynamic
 - molecular-dynamics
 - molecular-dynamic
-- namd
 - md trajectory
 - md trajectories
 - md simulation
 - md simulations
-- amber
 - gromacs
+- gromos
+- namd
+- amber
+- amber96
+- amber99
+- amber14
 - charmm
+- charmm27
+- charmm36
```

Comment:
same with charmm - maybe just charmm + charmm36 is enough?

Reply:
Yep